docs/docs/index.md

```diff
@@ -249,6 +248,19 @@ def report_failure(job, connection, type, value, traceback):
 Failure callbacks are limited to 60 seconds of execution time.
 
 
+### Stopped Callbacks
+
+Stopped callbacks are functions that accept `job` and `connection` arguments.
+
+```python
+def report_stopped(job, connection):
```

Comment:
Can you add an example of how to pass in `on_stopped` callback? Maybe put this on line 215.

Reply:
Added